src/main/java/trycb/service/Airport.java

```diff
@@ -1,32 +1,33 @@
 package trycb.service;
 
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
 import com.couchbase.client.core.error.QueryException;
 import com.couchbase.client.java.Cluster;
 import com.couchbase.client.java.json.JsonObject;
 import com.couchbase.client.java.query.QueryOptions;
 import com.couchbase.client.java.query.QueryResult;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.dao.DataRetrievalFailureException;
 import org.springframework.stereotype.Service;
-import trycb.model.Result;
 
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
+import trycb.model.Result;
 
 @Service
 public class Airport {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Airport.class);
 
-
     /**
      * Find all airports.
      */
     public static Result<List<Map<String, Object>>> findAll(final Cluster cluster, final String bucket, String params) {
         StringBuilder builder = new StringBuilder();
-        builder.append("select airportname from `").append(bucket).append("` where ");
+        builder.append("SELECT airportname FROM `").append(bucket).append("`.inventory.airport WHERE ");
```

Comment:
that's depressing syntax isn't it, does Java still not have string templates?
I wonder if would be clearer with the appends on newlines, but is fine.